cinema_service/settings.py

```diff
@@ -42,9 +40,11 @@
     "rest_framework",
     "cinema",
     "user",
+    "debug_toolbar",
```

Comment:
The `debug_toolbar` is included in `INSTALLED_APPS` and its middleware is active. This should be removed or conditionally included only in development environments to avoid performance overhead and potential security risks in production.